src/languageSupport/languages/influxql/connection.ts

```diff
@@ -178,7 +179,7 @@ export class ConnectionManager extends AgnosticConnectionManager {
       sessionCb,
       dispatch
     )
-    if (!shouldContinue) {
+    if (!shouldContinue || !isFlagEnabled('schemaComposition')) {
```

Comment:
Adding the feature flag `schemaComposition` to control the query composition effect. If the feature flag is off, users should explorer the schema list in the left sidebar without getting the auto query composition in the text editor.